pkg/components/imguploader/imguploader.go

```diff
@@ -110,8 +110,10 @@ func NewImageUploader() (ImageUploader, error) {
 		account_name := azureBlobSec.Key("account_name").MustString("")
 		account_key := azureBlobSec.Key("account_key").MustString("")
 		container_name := azureBlobSec.Key("container_name").MustString("")
+		sas_token_expiration := azureBlobSec.Key("sas_token_expiration").MustString("")
```

Comment:
I'd suggest either making this a `Duration` or `int`, so that we do the conversion in the same location as other variables.

If you don't want to do that, please add `_days` as the unit to the argument, since that'll be unclear otherwise.

```suggestion
		sas_token_expiration := azureBlobSec.Key("sas_token_expiration").MustDuration(0)
```

```suggestion
		sas_token_expiration := azureBlobSec.Key("sas_token_expiration_days").MustInt(-1)
```

Reply:
Added this:

```golang
sas_token_expiration := azureBlobSec.Key("sas_token_expiration_days").MustInt(-1)
```